ports/tbb/portfile.cmake

```
@@ -1,142 +1,30 @@
-if (NOT VCPKG_TARGET_IS_LINUX AND NOT VCPKG_TARGET_IS_OSX)
-    vcpkg_fail_port_install(ON_ARCH "arm" "arm64" ON_TARGET "uwp")
-endif()
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO oneapi-src/oneTBB
-    REF eca91f16d7490a8abfdee652dadf457ec820cc37 # 2020_U3
-    SHA512 7144e1dc68304b5358e6ea330431b6f0c61fadb147efa353a5b242777d6fabf7b8cf99b79cffb51b49b911dd17a9f1879619d6eebdf319f23ec3235c89cffc25
-    HEAD_REF tbb_2019
+    REF 46fb877ef1618d9de9a9ba10cee107592b7cdb2d # 2020_U3
+    SHA512 0ad688694e5d78d2266e804d9366465534af81051f345d8309ab69c6df0f74a92a341de799bdd72edab850a64f265df6b428225ef94468b26235aab2e0247747
+    HEAD_REF v2021.1.1
```

V2021.1.1 seems like a tag name instead of branch name. It might be master.
```suggestion
    HEAD_REF master
```